x/ccv/provider/keeper/relay_test.go

```diff
@@ -773,6 +774,12 @@ func TestEndBlockVSU(t *testing.T) {
 
 	testkeeper.SetupMocksForLastBondedValidatorsExpectation(mocks.MockStakingKeeper, 5, lastValidators, powers, -1)
 
+	sort.Slice(lastValidators, func(i, j int) bool {
+		return lastValidators[i].ConsensusPower(sdk.DefaultPowerReduction) >
```

Comment:
`ConsensusPower` versus `GetConsensusPower`?

Reply:
do the same thing, I changed to Get...